articles/quickstarts/search.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Q# quickstart: TODO"
+description: TODO
+author: cgranade
+ms.author: chgranad@microsoft.com
+ms.date: 10/9/2019
+ms.topic: article
+uid: microsoft.quantum.quickstarts.search
+---
+
+# Search for marked inputs with Grover's Algorithm
+
+This introduction uses [Grover's Algorithm][grover] to search inputs to a function for a particular _marked input_.
+This algorithm can be used to search for inputs with some special property, to find items in an unstructured list, and so forth.
+Grover's algorithm lets us use a quantum computer to run this search in less steps than the number of items in the list that we're searching — something no classical algorithm can do.
+
+In this introduction, we see how we can use Q# and the Quantum Development Kit to run Grover's algorithm with just a few lines of code.
+
+## Prerequisites
```

Comment:
We should also spell out what a user will learn from this quickstart, or what result they will get. There is a standard list format for this. I can help you out with it

Reply:
I'd be much obliged for your help on that, thank you!